boards/posix/native_posix/doc/board.rst

```diff
@@ -168,6 +168,23 @@ available options::
      [-no-rt]                  :Do NOT slow down the execution to real time, but
                                 advance Zephyr's time as fast as possible and
                                 decoupled from the host time
+     [-rt-drift=<drift>]       :Drift of the simulated clock relative to the
```

Comment:
Removing the (confusing) "parts per 1" and your added explanation works better.  thanks.